src/applications/edu-benefits/feedback-tool/helpers.js

```diff
@@ -68,6 +68,7 @@ function checkStatus(guid) {
       if (res instanceof Error) {
         Raven.captureException(res);
         Raven.captureMessage('vets_gi_bill_feedbacks_poll_client_error');
+        recordEvent({ event: 'edu-feedback-tool-submission-failed' });
```

Comment:
Should the prefix now be used to compute these strings?

Reply:
Good idea, updated.